roles/install-calico/tasks/calico-operator.yaml

```diff
@@ -0,0 +1,33 @@
+- name: Set up Calico - {{ calico_version }} using tigera operator.
+  command: kubectl create -f https://raw.githubusercontent.com/projectcalico/calico/{{ calico_version }}/manifests/tigera-operator.yaml
+  environment:
+    KUBECONFIG: /etc/kubernetes/admin.conf
+
+- name: Download Calico custom-resource manifest.
+  get_url:
+    url: "https://raw.githubusercontent.com/projectcalico/calico/{{ calico_version }}/manifests/custom-resources.yaml"
+    dest: /tmp/calico.yaml
+    mode: '0755'
+
+# Guide to set v-eth mtu for calico - https://projectcalico.docs.tigera.io/networking/mtu
+- name: Set veth_mtu to {{ calico_mtu }}
+  lineinfile:
+    path: /tmp/calico.yaml
+    insertafter: "calicoNetwork:"
+    line: "    mtu: {{ calico_mtu }}"
```

Comment:
lets add a TODO to have a better way to add a field into yaml spec

Reply:
Addressed.